### PR TITLE
Doc

### PR DESCRIPTION
**Pull request by daithiocrualaoich**
abaaead488e

### CODE REVIEW

File: docs/kubernetes.rst [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+itself.
+
+Minikube has a convenient subcommand for issuing the commands to create this
+container and to open the dashboard page in your default browser.
+
+::
+
+    minikube dashboard
+
+Note that if this command is run immediately after ``minikube start`` then it
```
Comment (passy) [inline docs/kubernetes.rst]: Stuff like this is so helpful. I wish more people would include that in their docs. Actually, can't you just write everyone's documentation?

File: docs/kubernetes.rst [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+
+.. _Kubernetes: https://kubernetes.io
+.. _cri-o: https://github.com/kubernetes-incubator/cri-o
+
+
+.. _kubernetes-tutorial:
+
+Tutorial
+~~~~~~~~
+_Minikube is a version of Kubernetes which runs a single node cluster in a VM.
```
Comment (passy) [inline docs/kubernetes.rst]: I *think* you meant to use `Minikube_` here.

File: docs/kubernetes.rst [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+::
+
+    kubectl delete service hello-minikube
+    kubectl delete deployment hello-minikube
+
+Currently, Kuberenetes can only pull images from Docker registries. This can
+make it awkward to load images into the rkt runtime. It is possible to run a
+Docker registry locally on the Kubernetes cluster but it doesn't function with
+the rkt runtime and has to be started separately with Docker on the Minikube
+instance. Even then, rkt errors on fetching images unless the registry is using
```
Comment (passy) [inline docs/kubernetes.rst]: Good rkt. Don't accept non-TLS.

File: docs/kubernetes.rst [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+    brew install kubectl
+
+.. _kubectl: https://kubernetes.io/docs/user-guide/kubectl-overview/
+
+Install the xhyve driver and set some permissions in order to use the native
+macOS hypervisor.
+
+::
+
+    brew install docker-machine-driver-xhyve
```
Comment (daithiocrualaoich) [inline docs/kubernetes.rst]: Yeah, I know, doggie. I think we all end up with the Docker HyperKit code when we want to use the macOS inbuilt hypervisor. :(

File: docs/kubernetes.rst (hunk 1 above)
Comment (daithiocrualaoich) [inline docs/kubernetes.rst]: I'm in awe of the http://www.writethedocs.org group. Their writing documentation guide is glorious.

File: docs/kubernetes.rst (hunk 3 above)
Comment (daithiocrualaoich) [inline docs/kubernetes.rst]: Well... you can flag the `rkt fetch` with `--insecure-options=http` and it's accept non-TLS as well as send your credentials in the clear and steal your dog. Other than the principle involved I wasn't going figuring out how to set the option in Kubernetes that turns on this flag in rkt calls. Might not even be possible.

https://coreos.com/rkt/docs/latest/commands.html#--insecure-options

**Pull request merged**